test/Microsoft.TestPlatform.AcceptanceTests/AppDomainTests.cs

```diff
@@ -54,13 +66,13 @@ private static bool IsFilesContentEqual(string filePath1, string filePath2)
             Assert.IsTrue(File.Exists(filePath2), "File doesn't exist: {0}.", filePath2);
             var content1 = File.ReadAllText(filePath1);
             var content2 = File.ReadAllText(filePath2);
-            Assert.IsTrue(string.Equals(content1, content2, StringComparison.Ordinal), "Content miss match file1 content:{2}{0}{2} file2 content:{2}{1}{2}", content1, content2, Environment.NewLine);
+            Assert.IsTrue(string.Equals(content1, content2, StringComparison.Ordinal), "Content mismatch{2}- file1 content:{2}{0}{2}- file2 content:{2}{1}{2}", content1, content2, Environment.NewLine);
             return string.Equals(content1, content2, StringComparison.Ordinal);
         }
 
         private string GetInProcDataCollectionRunsettingsFile(bool disableAppDomain)
         {
-            var runSettings = Path.Combine(Path.GetTempPath(), "test_" + Guid.NewGuid() + ".runsettings");
+            var runSettings = Path.Combine(GetTempPath(), "test_" + Guid.NewGuid() + ".runsettings");
```

Comment:
Possible improvement would be adding BannedApiAnalyzer to this project and banning System.IO.Path.GetTempPath.

Reply:
cool I vote for it

Reply:
Done.